x-pack/plugins/session_view/common/constants.ts

```diff
@@ -177,3 +179,9 @@ export const ALERT_FIELDS = PROCESS_EVENT_FIELDS.concat([
   'kibana.alert.rule.parameters.query',
   'kibana.alert.rule.query',
 ]);
+export const EVENT_ACTION_FORK: EventAction = 'fork';
+export const EVENT_ACTION_END: EventAction = 'end';
+// exec event.action is used by Endpoint and Cloud Defend
+export const EVENT_ACTION_EXEC: EventAction = 'exec';
+// executed event.action is used by Auditbeat
+export const EVENT_ACTION_EXECUTED: EventAction = 'executed';
```

Comment:
I'm not familiar with the event.action nomenclature , but if logically exec and executed are similar, we could rename the const to `EVENT_ACTION_EXEC_AUDITBEAT` to remove the need to add this comment everywhere

Reply:
I added the comment to clarify event action. For right now, we know Auditbeat event action is `executed`. When we will add Third Party integrations support for Session we don't know  whether EventActon could `exec` or `executed`.  I prefer to keep the event action name generic for now refactor if needed later.